src/cpu/aarch64/injectors/jit_uni_eltwise_injector.cpp

```diff
@@ -2398,8 +2398,7 @@ void jit_uni_eltwise_injector_f32<isa>::register_table_entries() {
     }
 }
 
-template struct jit_uni_eltwise_injector_f32<sve_512>;
-template struct jit_uni_eltwise_injector_f32<sve_256>;
+// We only need sve_128 as the injector is fully vector length agnostic
```

Comment:
If the point about vlen agnostic is true, maybe stripping a templated argument then?
At least, would be nice to consider and if considered, a TODO would be a good thing to have here.

Reply:
We are also exploring options to make this clearer, possibly by adding an `sve` enum value to signal VLA SVE. Or making the arch orthogonal to the SVE vector length. Either way, I think this is best done in an RFC, and we can come back to it